src/rgw/rgw_sal_motr.cc

```diff
@@ -2746,21 +2731,20 @@ int MotrObject::fetch_null_obj(const DoutPrefixProvider *dpp, bufferlist& bl)
   rc = this->fetch_null_obj_reference(dpp, prev_null_obj_key);
   if (rc < 0)
      return rc;
-
-  // Check entry in the cache
+  // 1st put, no null reference entry present in bucket.
+  if (prev_null_obj_key.empty())
+    return rc;
```

Comment:
and its better to log for this condition

Reply:
done